### PR TITLE
statements: Add SHOW BINARY LOG STATUS info

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Add info about `SHOW BINARY LOG STATUS` being a new alias for `SHOW MASTER STATUS`.

Related to https://github.com/pingcap/tidb/issues/48673

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: sql-statements/sql-statement-show-master-status.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ SHOW MASTER STATUS;
 
 The output of `SHOW MASTER STATUS` is designed to match MySQL. However, the execution results are different in that the MySQL result is the binlog location information and the TiDB result is the latest TSO information.
 
+The `SHOW BINARY LOG STATUS` statement was added in TiDB as alias for `SHOW MASTER STATUS` as that statement has been deprecated in MySQL 8.2.0 and newer.
```
Comment (ran-huang) [inline sql-statements/sql-statement-show-master-status.md:32]:
```suggestion
The `SHOW BINARY LOG STATUS` statement was added in TiDB as an alias for `SHOW MASTER STATUS`, which has been deprecated in MySQL 8.2.0 and newer versions. 
```

**Review by ran-huang — APPROVED** [review]

**Comment by ran-huang** [comment]
@dveeden Given the merge date of the pr (https://github.com/pingcap/tidb/pull/48678), I assume the new alias is not included in v7.5, right?

**Comment by dveeden** [comment]
> @dveeden Given the merge date of the pr ([pingcap/tidb#48678](https://github.com/pingcap/tidb/pull/48678)), I assume the new alias is not included in v7.5, right?

Yes correct.

> @dveeden Given the merge date of the pr ([pingcap/tidb#48678](https://github.com/pingcap/tidb/pull/48678)), I assume the new alias is not included in v7.5, right?

Yes correct. This is not in v7.5.

```
sql> SHOW MASTER STATUS;
+-------------+--------------------+--------------+------------------+-------------------+
| File        | Position           | Binlog_Do_DB | Binlog_Ignore_DB | Executed_Gtid_Set |
+-------------+--------------------+--------------+------------------+-------------------+
| tidb-binlog | 445788609072857088 |              |                  |                   |
+-------------+--------------------+--------------+------------------+-------------------+
1 row in set (0.0003 sec)

sql> SHOW BINARY LOG STATUS;
ERROR: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 11 near "BINARY LOG STATUS" 

sql> SELECT TIDB_VERSION()\G
*************************** 1. row ***************************
TIDB_VERSION(): Release Version: v7.5.0-alpha-258-g603b47c991-dirty
Edition: Community
Git Commit Hash: 603b47c9917af415264f3de70359abadba2cd5bb
Git Branch: release-7.5
UTC Build Time: 2023-11-21 06:34:37
GoVersion: go1.21.0
Race Enabled: false
Check Table Before Drop: false
Store: unistore
1 row in set (0.0006 sec)
```

**Pull request merged**